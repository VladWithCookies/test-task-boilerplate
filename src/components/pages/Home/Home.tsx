import MainLayout from 'components/templates/MainLayout';

export default function Home() {
  return (
    <MainLayout>
      Welcome to the Test Task Boilerplate
    </MainLayout>
  );
}
