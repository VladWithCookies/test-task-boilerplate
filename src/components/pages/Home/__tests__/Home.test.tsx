import { render } from 'utils/tests/renderer';

import Home from '../Home';

describe('Home', () => {
  it('matches snapshot', () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
