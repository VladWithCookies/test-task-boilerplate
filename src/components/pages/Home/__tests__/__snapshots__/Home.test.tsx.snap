// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches snapshot 1`] = `
<div>
  <div
    class="chakra-container css-0"
  >
    <div
      class="chakra-stack css-0"
    >
      <div
        class="css-0"
      >
        <a
          href="/"
        >
          <h1
            class="chakra-heading css-0"
          >
            Test Task Boilerplate
          </h1>
        </a>
      </div>
    </div>
  </div>
</div>
`;
